prow/plugins/trigger/generic-comment.go

```diff
@@ -93,6 +79,25 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 		}
 	}
 
+	baseSHA, presubmits, err := getPresubmitsForPR(c, pr)
+	if err != nil {
+		return err
+	}
+
+	// Skip comments not germane to this plugin
```

Comment:
Why did this get moved? It should sit at the top to reduce API calls

Reply:
(can we load the repo presubmits before all the other stuff)?

Reply:
The reason for moving it is that I initially used clonerefs to get the repo cloned instead of `prow/git`. That made it a pretty expensive operation, so I moved it to the end.

Since `prow/git` caches the cloning should not be expensive anymore so shouldn't mater. Should I move it back?

Reply:
Expense in time and CPU load is much easier to eat than expense in API tokens for GitHub, so I would think we want this frst